testdata/server.crt

```diff
@@ -1,19 +1,20 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
We should probably auto generate this when the tests start up. We have all the tooling to do this in CFSSL.